docs/http/compliance-matrix.md

```diff
@@ -0,0 +1,65 @@
+# Compliance of implementations with stable HTTP semantic conventions
+
+The following tables show the compliance of HTTP instrumentation published under the OpenTelemetry GitHub org
+with the stable HTTP semantic conventions.
+
+`+` means the feature is supported, `-` means it is not supported, `N/A` means
+the feature is not applicable to the particular instrumentation, blank cell means the
+status of the feature is not known.
+
```

Comment:
Should we maybe also describe here or at least link to the definition / meaning of `OTEL_SEMCONV_STABILITY_OPT_IN` somewhere here?